sdk/appconfiguration/pom.service.xml

```diff
@@ -9,10 +9,9 @@
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
-    <!-- NOTE: Commenting out because we should now be binary composed. 
+    <!-- NOTE: Commenting out because we should now be binary composed.
```

Comment:
Does this comment apply? We're still compiling from modules.

Reply:
Engsys have aspirations here, so I'm leaving it in for now.